ppl/cmd/zar/README.md

```diff
@@ -32,26 +32,24 @@ ln -s zq-sample-data/zng
 
 ## ingesting the data
 
-Let's take those logs and ingest them into a directory.   We'll make it
-easier to run all the commands by setting an environment variable pointing
-to the root of the logs tree.
+Let's take those logs and ingest them into a directory.  Often we'd keep our
+archive somewhere like [Amazon S3](https://aws.amazon.com/s3/), but since we're
+just doing a quick test we'll use local temp space.  We'll make it easier to
+run all the commands by setting an environment variable pointing to the root of
+the logs tree.
 ```
-mkdir ./logs
-set ZAR_ROOT=`pwd`/logs
+mkdir /tmp/logs
+ZAR_ROOT=/tmp/logs
 ```
 
 Now, let's ingest the data using "zar import".  We are working on more
 sophisticated ways to ingest data (e.g., by the standard time partitioning
 techniques of year/month/day/hour etc), but for now zar import just chops
-its input into chunk files of approximately equal size.
-
-Zar import expects its input to be
-in the zng format so we'll use zq to take all the zng logs, gunzip them,
-and feed them to "zar import", which here expects its data on stdin.  We'll chop them
-into chunks of 25MB, which is very small, but in this example the data set is
+its input into chunk files of approximately equal size. We'll chop into chunks
+of 25MB, which is very small, but in this example the data set is
 fairly small (175MB) and you can always try it out on larger data sets:
 ```
-zq zng/*.gz | zar import -s 25MB -
+zar import -s 25MB zng/*.gz
```

Comment:
I had a [Slack thread](https://brim-labs.slack.com/archives/CP3BMAEEP/p1614885363079800) with @alfred-landrum where I confirmed that this change was appropriate to make. I paused at the time because moving from the prior command line to the newer/simpler one actually changed how the events were distributed across the chunks, and I wanted to make sure that was expected and not a bug. His take at the time:

> It's not a problem, but we should change the readme to use the `zar import <file> ...` since that's the simpler usage. I believe the difference between the two ways is that when `zq` is given multiple files, it tries to merge them into a single stream by looking for a `ts` field, and assumes they are in ascending order, whereas `zar import` does something similar, but assumes the files are using the same order as the archive (which defaults to descending).